### PR TITLE
Version bump of 1.2.0 

### PR DESCRIPTION
**Pull request by chefsale**
This is using the automatic toml update of the version and it unwrapped the dependencies as well. In the future only versions should be updated.

### CODE REVIEW

File: neard/Cargo.toml [hunk 1]
```diff
@@ -1,7 +1,15 @@
+[[bin]]
+path = "src/main.rs"
+name = "neard"
+
+[[bin]]
+path = "src/../src/main.rs"
```
Comment (bowenwang1996) [inline neard/Cargo.toml:6]: why?

Comment (chefsale) [reply neard/Cargo.toml:6]: That's on master for a while like that:
https://github.com/nearprotocol/nearcore/blob/master/neard/Cargo.toml#L58

**Comment by chefsale** [comment]
> why not fix the automated tool?

The automated tool is actually doing the right thing:

This is the code:
```
        cargo_toml = toml.loads(base64.b64decode(cargo.content).decode("utf-8"))
        previous_version = cargo_toml['package']['version']
        if cargo_toml['package']['version'] == version:
            logging.info("Versions are the same. Skipping bumping")
            return

        cargo_toml['package']['version'] = version
        response = self.repo.update_file(
            "neard/Cargo.toml",
            f"Automatic version bump from {previous_version} to {version}",
            toml.dumps(cargo_toml), cargo.sha, branch_name
```

dumps just spits out a string which is minimized, couldn't find a way to specify not to do that. we could create some custom parser, but this way I thought we would be more consistent and precise.

**Comment by frol** [comment]
Can the tool keep the existing formatting of the file and only touch the relevant piece? (I know it might be challenging, but still worth asking as it is common to have `package` section on the top while it seems that it rewrote the whole file to its liking.

**Comment by chefsale** [comment]
Yep, will do that.

**Comment by chefsale** [comment]
@frol @bowenwang1996 Here you go, its a dirty hack, but it works: 
https://github.com/nearprotocol/near-ops/pull/74/commits/ba047d3377915a322a256f06d51ab535db047fc3

I tried with regexes first, but this seemed like a better simple way of not messing up.

**Pull request merged**